content/docs/concepts/components/buildpack.md

```diff
@@ -5,33 +5,34 @@ weight=2
 
 ## What is a buildpack?
 
-A buildpack is a unit of work that inspects your app source code and formulates a plan to build and run your
+A buildpack is a set of scripts that inspects your app source code and creates a plan to build and run your
 application.
 
 <!--more-->
 
-Typical buildpacks are a set of at least three files:
+Typical buildpacks consist of at least three files:
 
 * `buildpack.toml` -- provides metadata about your buildpack
 * `bin/detect` -- determines whether buildpack should be applied
-* `bin/build` -- executes buildpack logic
+* `bin/build` -- executes build logic
 
 #### Meta-buildpack
 
 There is a different type of buildpack commonly referred to as a **meta-buildpack**. It contains only a 
 `buildpack.toml` file with an `order` configuration that references other buildpacks. This is useful for 
 composing more complex detection strategies.
 
-## Anatomy of a buildpack
+## Buildpack phases
 
-There are two essential phases that allow buildpacks to create a runnable image.
+These are two essential steps that allow buildpacks to create a runnable image.
 
 #### Detect
 
-A platform sequentially tests groups of buildpacks against your app's source code. The first group that deems itself 
-fit for your source code will become the selected set of buildpacks for your app. Detection criteria is specific to each 
-buildpack -- for instance, an **NPM buildpack** might look for a `package.json`, and a **Go buildpack** might look for 
-Go source files.
+To determine which buildpack should be used to build your app, a [`platform`](platform) sequentially tests a group of
+buildpacks against your source. The first buildpack that claims that it can build your sources will become selected.
```

Comment:
This may not always be true. The first buildpack group is selected but there could possibly be multiple buildpacks in that group that can build your source code.

Reply:
On Heroku I only used one buildpack for one build. This group logic is something new. It deserves a separate paragraph to keep it simple. Maybe it is documented somewhere already? How buildpacks share results of the build? Who creates the final image and packs artifacts then?

Reply:
They all work together to create the output image. See https://buildpacks.io/docs/reference/spec/buildpack-api/#build-plan

Reply:
I am not too sure about how heroku uses CNB but I think they also allow multiple buildpacks https://devcenter.heroku.com/articles/using-multiple-buildpacks-for-an-app

Reply:
@samj1912 it looks like there is no obligation for the `buildpack` to provide build plan. And even if it provides, it just to pass info from own `detect` to own `build`, and further orchestration of buildpacks is the role of `lifecycle` tool, so I think it is okay to omit this here.

Reply:
I agree with @samj1912, this sentence may be very misleading:

> The first buildpack that claims that it can build your sources will become selected.

It's really more like the "first group of buildpacks that claim they can..."

Reply:
@jromero I am lost. How can a "group of buildpacks" collectively determine if they fit? Is it still `platform` that selects multiple buildpacks based on their individual responses? Or what is the "consensus algorithm" that makes buildpacks determine that they can only do this build together?

It is possible to document the simple case and then adjust for a more complicated scenario, which I believe needs a separate page.

Reply:
Sorry for the delay. You can learn more about how buildpacks groups work here: https://buildpacks.io/docs/concepts/components/buildpack-group/

I agree with the sentiment that we could document a simple case and add more complicated scenarios later but the sentence in question is just wrong based on the existing functionality. 

A simple change from:

> The first buildpack that claims that it can build your sources will become selected.

To:

> The first group of buildpacks that claim to be able to build your sources will be used.

Should resolve the apprehension.

Reply:
Looking back at was originally there, I think that makes good enough sense. If the reader needs more clarification, then maybe we can refer them to https://buildpacks.io/docs/concepts/components/buildpack-group/.